virtool/samples/tasks.py

```diff
@@ -51,3 +51,18 @@ def __init__(self, task_id, data, context, temp_dir):
 
     async def deduplicate_sample_names(self):
         await self.data.samples.deduplicate_sample_names()
+
+
+class UpdateWorkflowsFieldsTask(BaseTask):
+    """
+    Updates workflows, nuvs, and pathoscoope fields for samples
+    """
+
+    name = "populate_workflows_field"
```

Comment:
The class name and `name` class attribute should match up.

Let's call it `UpdateSampleWorkflowsTask` and `update_sample_workflows`.